app/helpers/tags_helper.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module TagsHelper
+  # rubocop:disable MethodLength
+  def tag_usage(data_id, data_type, user_id)
+    result = []
+    moments = Moment.where(user_id: user_id).order('created_at DESC')
+    if data_type == 'category'
+      strategies = Strategy.where(user_id: user_id).order('created_at DESC')
+      [moments, strategies].each do |records|
```

Comment:
can both branches of this conditional be extracted to private methods that return the proper array and assign that to the `result` variable?
so something like:
```
if data_type == 'category'
   result  = get_moments
elsif data_type.in?(%w[mood strategy])
   result = get_moods
end
```
or something along those lines, maybe not those method names specifically but that's the idea